project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_members/c4m_features_og_members.module

```diff
@@ -516,12 +516,21 @@ function _c4m_features_og_members_get_user_type($group = NULL, $user = NULL) {
     return C4M_USER_TYPE_GO;
   }
 
-  // The user has the group role 'administrator member'.
-  if (in_array('administrator member', og_get_user_roles('node', $group->nid, $user->uid))) {
-    return C4M_USER_TYPE_GA;
-  }
+  if ($group->type == 'group') {
```

Comment:
the condition should only check if it's 'project', no need the `else` afterward ...